pages/mesosphere/dcos/1.13/release-notes/1.13.5/index.md

```diff
@@ -27,9 +27,12 @@ DC/OS is a distributed operating system that enables you to manage resources, ap
 - Marathon: Fixed a bug in which a service could get stuck if a failure occurred while Mesos tried to create a reservation. (MARATHON-8693) 
 - [enterprise]Updated `dcos-backup` to support accessing Exhibitor through Admin Router. This is needed when [Exhibitor mutual TLS authentication](https://docs.d2iq.com/mesosphere/dcos/1.13/security/ent/tls-ssl/exhibitor/) is enabled. (DCOS-57704)[/enterprise]
 - Metronome: Post-install configuration can now be added to `/var/lib/dcos/metronome/environment`. (DCOS_OSS-5509)
-- Mesos overlay networking: Added an HTTP endpoint for dropping agents from the state.  (DCOS_OSS-5536)
-- Admin Router: Improved service routing robustness by omitting Marathon apps with wrongly specified `DCOS_SERVICE_PORT_INDEX` values. (DCOS_OSS-5491)
-- Strict volume name validation was not relaxed enough in DC/OS release 1.13.4; this has been resolved. (MARATHON-8697)
+- Mesos overlay networking: Added an HTTP endpoint for dropping agents from the state.  (DCOS_OSS-5536, COPS-5281)
+- Admin Router: Improved service routing robustness by omitting Marathon apps with wrongly specified `DCOS_SERVICE_PORT_INDEX` values. (COPS-5147, DCOS_OSS-5491)
+- Strict volume name validation was not relaxed enough in DC/OS release 1.13.4; this has been resolved. (MARATHON-8697, MARATHON-8681, COPS-5219)
+- Mesos: Fixed race condition between two terminal task status updates for Docker/Command executor. (COPS-4995, MESOS-9887)
+- Fixed a problem with the `dcos-iam-ldap-sync` service failing to start correctly after a system reboot (COPS-4814)
```

Comment:
This entry needs to be removed again.

This would be a false positive: this is *not* a change that happened for the 1.13.5 release, but a change that was made between DC/OS Enterprise 1.13.0 and 1.13.1.

Reply:
Done.